sisl/grid.py

```diff
@@ -179,6 +179,84 @@ def interp(self, shape, order=1, mode="wrap", **kwargs):
 
         # Apply the scipy.ndimage.zoom function and return a new grid
         return self.apply(ndimage_zoom, zoom_factors, mode=mode, order=order, **kwargs)
+    
+    def transform_axes(self, axes=np.eye(3), output_shape=None, mode="constant", order=1, **kwargs):
+        """
+        Applies a linear transformation to the grid to get it relative to arbitrary axes.
+
+        This method can be used, for example to get the values of the grid with respect to
+        the standard basis, so that you can easily visualize it or overlap it with other grids
```

Comment:
"standard basis" -> "cartesian basis"?

Reply:
Yes, probably. I used standard basis since it is how it's called in math and that's all the method does (math). But going back to the physics world it probably makes more sense.